README.md

```diff
@@ -381,63 +387,164 @@ information, see <https://tools.ietf.org/html/rfc6749>.
-also be used with an existing CA. Here are the instructions to use create_certificates
-with an existing CA.
-
-1. To configure the create_certificates.py script, use create_certificates.ini. In
-   create_certificates.ini, set `ca_private_key_path` and `ca_certificate_path` to the
-   path of your CA private key and certificate. Configure the remaining parameters as
-   desired. The SAN (subject alternative name) parameters will need to be set to the
-   appropriate IP addresses and DNS names of your server and client.
-
-1. While in scos-sensor root directory, run the create_certificates.py script passing
-   the following arguments in the listed order:
-
-   - ini_path - path to the create_certificates.ini file.
-   - ini_section - section of the INI file to use.
-   - key_passphrase - Passphrase to use to encrypt private keys. Set to `None` to
-     disable encryption.
-
-   The following certificates will be generated:
-
-   - sensor01_private.pem - sensor private key.
-   - sensor01_certificate.pem - sensor certificate.
-   - sensor01_client_private.pem - client private key.
-   - sensor01_client.pem - client certificate.
-
-1. Copy sensor01_private and sensor01_certificate to the computer where the scos-sensor
-   will run. If you are using client certificates, also copy the CA certificate used to
-   generate the certificates. Make sure the certificates are somewhere in configs/certs,
-   and that `SSL_CERT_PATH` and `SSL_KEY_PATH` (in the environment file) are set to the
-   paths of the certificates relative to configs/certs. If you are using client
-   certificates, set `SSL_CA_PATH` to the path of the CA certificate relative to
-   configs/certs.
-
-1. Run scos-sensor. If you are using client certificates, use
-   sensor01_client_private.pem and sensor01_client to connect to the API.
-
-The create_certificates.py script can also generate a new CA and use it for generating
-the certificates. To run create_certificates.py this way, comment out
-`ca_private_key_path` and `ca_certificate_path` in create_certificates.ini, make sure
-`ca_private_key_save_path` and the other parameters are set as desired, then repeat
-steps 2-4 above. The CA private key file (saved to ca_private_key_save_path) and the CA
-public key (scostestca.crt) will be generated in addition to the files listed in step
-2 above.
+This section describes how to create a self-signed root CA, SSL server certificates for
+the sensor, optional client certificates, and test JWT public/private key pair.
+
+As described below, a self-signed CA can be created for testing. **For production, make
+sure to use certificates from a trusted CA.**
+
+Below instructions adapted from
+[here](https://www.golinuxcloud.com/openssl-create-client-server-certificate/#OpenSSL_create_client_certificate).
+
+##### Sensor Certificate
+
+This is the SSL certificate used for the scos-sensor web server and is always required.
```

Comment:
Add a comments at the start that clarifies the relationship between these commands and the script referenced above.